jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -149,7 +149,7 @@ def test_spectrum3d_no_wcs_parse(cubeviz_helper):
     data = cubeviz_helper.app.data_collection[0]
     flux = data.get_component('flux')
     assert data.label.endswith('[FLUX]')
-    assert data.shape == (3, 2, 4)  # y, x, z
+    assert data.shape == (2, 3, 4)  # y, x, z
```

Comment:
```suggestion
    assert data.shape == (2, 3, 4)  # x, y, z
```
95% sure the comment was/is wrong.